doc/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,320 @@
+----
+
+=== Generate some Memory Load for HPA
+
+Let's have a look `TARGETS` part of HPA output
+
+[source, shell]
+----
+$ kubectl get hpa heap-based-hpa
+NAME             REFERENCE               TARGETS     MINPODS   MAXPODS   REPLICAS   AGE
+heap-based-hpa   StatefulSet/hazelcast   180m/200m   3         10        10         11m
+----
+
+As we see, current HPA Target is 180m/200m so if we increase memory usage just 10%  by adding 10MB into the cluster, HPA should trigger a scale up event. I will use Hazelcast Java Client to put some data into cluster, but you can use your own language to implement the same functionality. You can see all Hazelcast supported programming languages https://hazelcast.org/imdg/clients-languages/[here].
+
+Let's first port forward from our local machine to be able to connect remote k8s hazelcast member pod.
+
+[source, shell]
+----
+$ kubectl port-forward hazelcast-0 5701
+----
+
+Execute following code snippet to put data into Hazelcast Cluster.
+
+[source, java]
+----
+ // start Hazelcast Client with smartRouting enabled
+ ClientConfig cfg = new ClientConfig();
+ cfg.getNetworkConfig().setSmartRouting(false);
+ HazelcastInstance client = HazelcastClient.newHazelcastClient(cfg);
+----
+
+[NOTE]
+====
+You can use the client application from xref:kubernetes-external-client:ROOT:index.adoc[External Smart Client] guide to connect to Hazelcast cluster.
+====
+
+When you start putting data into your hazelcast cluster, you will see that new pods will be created and added to Hazelcast Cluster.
+
+[source, shell]
+----
+$ kubectl get po
+NAME                    READY   STATUS    RESTARTS   AGE
+hazelcast-0             1/1     Running   0          73m
+hazelcast-1             1/1     Running   0          72m
+hazelcast-2             1/1     Running   0          72m
+hazelcast-3             1/1     Running   0          16m
+hazelcast-4             1/1     Running   0          3m
+hazelcast-5             1/1     Running   0          3m
+hazelcast-6             1/1     Running   0          47s
+hazelcast-7             1/1     Running   0          41s
+hazelcast-8             1/1     Running   0          14s
+hazelcast-9             1/1     Running   0          13s
+hazelcast-mancenter-0   1/1     Running   0          73m
+----
+
+== Conclusion
+
+Autoscaling is an important feature for enterprises to save money and to cope with unexpected traffic coming to your deployments. However, configuring autoscaling needs to be done carefully because you might end up unnecessary scale up/down operations which might cost some instability in your system. In this blog post, we explained how you can use HPA with your Hazelcast Cluster based on Resource Metrics and Custom Metrics. If Kubernetes Pod/Node Level Cpu/Memory usage is fine for you then use Resource Metrics. If you have more specific requirements, and you need to have Hazelcast specific autoscaling capabilities, Custom Metrics is the answer.
```

Comment:
seems like this is written before guides platform comes in :)
```suggestion
Autoscaling is an important feature for enterprises to save money and to cope with unexpected traffic coming to your deployments. However, configuring autoscaling needs to be done carefully because you might end up unnecessary scale up/down operations which might cost some instability in your system. In this guide, we explained how you can use HPA with your Hazelcast Cluster based on Resource Metrics and Custom Metrics. If Kubernetes Pod/Node Level Cpu/Memory usage is fine for you then use Resource Metrics. If you have more specific requirements, and you need to have Hazelcast specific autoscaling capabilities, Custom Metrics is the answer.
```